tailwind.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
```

Comment:
I did not define them since it was mentioned about customizing them to individual use. I have tried to use the standard class namings provided on tailwind CSS. I will incorporate them once the backend is completed.